assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+          "mappings": [],
+          "freeform": {
+            "key": "charge"
+          },
+          "condition": {
+            "and": [
+              "fee=yes"
+            ]
+          }
+        },
+        {
+          "render": {
+            "en": "power supply"
+          },
+          "question": {
+            "en": "Does this place have a power supply?"
+          },
+          "freeform": {
+            "key": "power_supply",
+            "addExtraTags": [
+              "power_supply=yes"
+            ]
+          },
+          "condition": {
+            "and": []
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "power_supply=yes"
+                ]
+              },
+              "then": {
+                "en": "This place has a power supply"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "power_supply=no"
+                ]
+              },
+              "then": {
+                "en": "This place does not have power supply"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "water point"
+          },
+          "question": {
+            "en": "Does this place have a water point?"
+          },
+          "freeform": {
+            "key": "water_point",
+            "addExtraTags": [
+              "water_point=yes"
+            ]
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "water_point=yes"
+                ]
+              },
+              "then": {
+                "en": "This place has a water point"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "water_point=no"
+                ]
+              },
+              "then": {
+                "en": "This place does not have a water point"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "grey water"
+          },
+          "question": {
+            "en": "Can you dispose of grey water here?"
+          },
+          "freeform": {
+            "key": "sanitary_dump_station:grey_water"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station:grey_water=yes"
+                ]
+              },
+              "then": {
+                "en": "You can dispose of grey water here"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station:grey_water=no"
+                ]
+              },
+              "then": {
+                "en": "You cannot dispose of gray water here"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "chemical toilet"
```

Comment:
As usual - I know the custom theme generator is buggy...

Reply:
done